content/kubeflow/distributed.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Kubeflow Distributed Training"
+date: 2019-11-15T00:00:00-08:00
+weight: 70
```

Comment:
Let's push this to 90 for now since it has limited instructions

Reply:
This is much easier and user get less chance to meet problems.. I am not sure any instruction we need here?

Reply:
OK. Let me change to 90 for now.